src/tabcenter.js

```diff
@@ -8,10 +8,7 @@ function TabCenter() {
 
 TabCenter.prototype = {
   async init() {
-    const darkTheme = (await browser.storage.local.get({
-      darkTheme: false
-    })).darkTheme;
-    this.toggleTheme(darkTheme);
+    this._detectDarkTheme();
```

Comment:
Maybe add a comment here precising that this is done in the background on purpose. (you're not awaiting on the promise)